### DISCUSSION TITLE
unable to remove license from formatted phones

### DISCUSSION THREAD

**Discussion by gioelebianchi00** [mtotschnig/MyExpenses#1580] [Q&A]
Hi.
i've bought the extended license. my email is: gioele.bianchi@scoprimantova.it
i've exceeded the limit of licensed devices. 
the problem is that they aren't real devices, but i've formatted my phone more times and as well i connected the license.
how can i remove license from previous sessions? is there any license manager?
cheers

**Comment by mtotschnig** [comment]
Licence manager speaking :-) This is explained in the FAQ: https://github.com/mtotschnig/MyExpenses/wiki/FAQ:-Distribution#when-trying-to-validate-a-licence-key-on-the-f-droid-version-i-get-device-limit-exceeded
Please send me a message as explained there, so that I can validate your request.
